handbook/business-operations/README.md

```diff
@@ -828,6 +828,11 @@ When the final signature is added to an envelope in DocuSign, it is marked as co
 
 Domain name registrations are handled through Namecheap. Access is managed via 1Password.
 
+#### Vetty
+
+Fleet employees with access to Fleet's infrastructure undergo background check provided through Vetty.  
```

Comment:
- change Fleet employees to "Fleet team members"
- typo fix (a background check)
- add link for convenience

`Fleet team members with access to Fleet's infrastructure undergo a background check provided through [Vetty](https://vetty.co/).`